packages/console/src/types/theme.ts

```diff
@@ -1,4 +1,10 @@
+import { z } from 'zod';
+
 export enum Theme {
-  LightMode = 'light',
-  DarkMode = 'dark',
+  Light = 'light',
+  Dark = 'dark',
 }
+
+export const appearanceModeGuard = z.nativeEnum(Theme).or(z.literal('system'));
+
+export type AppearanceMode = z.infer<typeof appearanceModeGuard>;
```

Comment:
be better to define two enums and union them? since the sudden `z.literal()` looks weird